lib/src/wip/autocomplete/popup.dart

```diff
@@ -38,7 +41,7 @@ class _PopupState extends State<Popup> {
   void initState() {
     widget.controller.addListener(rebuild);
     this.width = 300;
```

Comment:
Paired variables like `width` and `height` should be processed alike. If you drop `this.` on one, please do so on the other too even though it is beyond the min required change.

Reply:
Done.